script-HW8.js

```diff
@@ -0,0 +1,66 @@
+// HW-JS8 //
+
+// Ex-1 //
+
+let student = {
+    firstName: "John",
+    lastName: "Doe",
+    faculty: "history",
+    year: 3
+};
+
+let studentInfo = ({firstName, lastName, fullName = `${firstName} ${lastName}`, faculty, year}) => {
```

Comment:
it's not the best practice to add logic inside the props declaration. Even though it is possible, it makes your code look messy. So I suggest you to destructure the `{firstName, lastName, faculty, year}`. And create a `fullName` inside the function.

Reply:
done